tests/common_vision_utils.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import json
+
+import shap
+from tensorflow.keras.applications.resnet50 import ResNet50, preprocess_input
+
+
+def load_imagenet_dataset():
+    X, _ = shap.datasets.imagenet50()
+    # load just the first 10 images
```

Comment:
Is this intentional?

Reply:
yes, just taking a sample for the test to reduce execution time of test